const.go

```diff
@@ -44,9 +44,9 @@ const SignedArmorString = "SIGNED MESSAGE"
 // DetachedSignatureArmorString is included in armor headers for detached signatures.
 const DetachedSignatureArmorString = "DETACHED SIGNATURE"
 
-// SaltpackFormatName is the publicly advertised name of the format,
-// used in the header of the message and also in Nonce creation.
-const SaltpackFormatName = "saltpack"
+// FormatName is the publicly advertised name of the format, used in
+// the header of the message and also in Nonce creation.
+const FormatName = "saltpack"
```

Comment:
Possible pre-existing problem: I don't think we check the format name during decryption. Should we be doing that? I guess it depends on whether we expect someone to change the format name as a sort of context string (where accidental decryption compatibility could be a security issue) or as part of extending it in their own application (where it might be a jerk move to break them).